### PR TITLE
EPMRPP-38631 - Webpack 4

### PR DESCRIPTION
**Pull request by mykhailo-kalashnikov-epam**

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
-    "build": "webpack -p --env.production",
-    "storybook": "start-storybook -p 9001 -c .storybook",
-    "storybook:build": "build-storybook -c .storybook -o .storybook_app",
+    "dev": "NODE_ENV=development webpack-dev-server  --port 3000",
+    "build": "webpack -p",
     "test": "jest --notify",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "build:lang:en": "webpack --display none --env.dev && babel localization/translator.js | node",
     "manage:translations": "webpack --display none --env.dev && babel-node ./localization/translatorRunner.js",
```
Comment (AmsterGet) [inline app/package.json]: Please make sure that the translations scripts will work properly.

File: app/package.json [hunk 2]
```diff
@@ -55,6 +55,7 @@
     "rc-slider": "8.6.3",
     "react": "16.6.0",
     "react-autosuggest": "9.3.4",
+    "react-c3js": "0.1.20",
```
Comment (AmsterGet) [inline app/package.json]: What is the purpose of this?

File: app/package.json [hunk 3]
```diff
@@ -10,18 +10,18 @@
     "format:js": "npm run eslint -- --fix",
     "format:scss": "npm run stylelint -- --fix",
     "format": "npm run format:js && npm run format:scss && prettier --write */**/*.md",
-    "dev": "webpack-dev-server --port 3000",
-    "build": "webpack -p --env.production",
-    "storybook": "start-storybook -p 9001 -c .storybook",
-    "storybook:build": "build-storybook -c .storybook -o .storybook_app",
+    "dev": "NODE_ENV=development webpack-dev-server  --port 3000",
```
Comment (AmsterGet) [inline app/package.json]: Why not `set NODE_ENV=development && webpack-dev-server  --port 3000`? For now it doesn't work on windows.

File: app/package.json [hunk 4]
```diff
@@ -10,18 +10,18 @@
...
     "test": "jest --notify",
     "test:watch": "jest --watch",
     "test:coverage": "jest --coverage",
     "build:lang:en": "webpack --display none --env.dev && babel localization/translator.js | node",
     "manage:translations": "webpack --display none --env.dev && babel-node ./localization/translatorRunner.js",
     "manage:translations:test": "webpack --display none --env.dev && babel-node ./localization/translatorRunnerCheck.js",
     "precommit": "lint-staged",
-    "postcommit": "git update-index --again"
+    "postcommit": "git update-index --again",
+    "storybook": "NODE_ENV=development start-storybook -p 9001 -c .storybook",
```
Comment (AmsterGet) [inline app/package.json]: `set NODE_ENV=development &&`

File: app/.babelrc [hunk 5]
```diff
@@ -1,25 +1,16 @@
...
-  ],
-  "plugins": [
-    "react-hot-loader/babel",
-    "transform-runtime",
-    "transform-decorators-legacy",
-    "transform-class-properties",
-    "transform-object-rest-spread",
+    "@babel/preset-react",
     [
-      "react-intl",
```
Comment (AmsterGet) [inline app/.babelrc]: This plugin has been used for extracting messages from files. Where is it being replaced?

**Comment by DiscoElevator** [comment]
After a short discussion, decided that ESLint rules are out of scope of this PR since turning them on require additional changes to the codebase.

**Comment by DiscoElevator** [comment]
Closed in favor of #2211

**Pull request closed without merging**